hello-world.md

```diff
@@ -290,7 +290,7 @@ Let's go over the cycle again
 
 On the face of it this may seem tedious but sticking to the feedback loop is important.
 
-Not only does it ensure that you have _relevant tests_ it helps ensure _you design good software_ by refactoring with the safety of tests.
+Not only it does ensure that you have _relevant tests_, it helps ensure _you design good software_ by refactoring with the safety of tests.
```

Comment:
I think the addition of the comma is correct, but 

> Not only it does ensure 

Isn't correct. Could you edit this particular bit back and then I'll merge the rest.